projects/canopy/src/assets/icons/pause-spot.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32"><defs><path id="a" d="M250.09 1C112.63 1 1.2 112.43 1.2 249.89s111.43 248.89 248.89 248.89 248.89-111.43 248.89-248.89S387.55 1 250.09 1zm-18.27 352.79h-77.64V146h77.64v207.79zm114.18 0h-77.64V146H346v207.79z"/></defs><use fill-rule="evenodd" xlink:href="#a"/></svg>
```

Comment:
Did you get this from a designer?

Reply:
Yes they were designed by Paul Farrignton